ci/meta.yml

```diff
@@ -24,6 +24,134 @@ meta:
         outputs:
           - name: terraform-config
       params:
-        AWS_ACCESS_KEY_ID: ((dataworks-secrets.ci_aws_access_key_id))
-        AWS_SECRET_ACCESS_KEY: ((dataworks-secrets.ci_aws_secret_access_key))
+        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
```

Comment:
do you need this, can you use the instance profile

Reply:
this is crown concourse.